src/containers/Elements/NodeProviderPanels.js

```diff
@@ -45,7 +44,7 @@ const getPanelConfigs = createSelector(
 );
 
 const getHighlight = (provider, panelNumber) => {
-  if (provider.highlight) { return provider.highlight; }
+  if (provider.highlight) { return colorDictionary[provider.highlight]; }
```

Comment:
What happens when `colorDictionary[provider.highlight]` isn't defined?

Reply:
Should return `colorDictionary['panel-highlight']` in that case. It doesn't exist right now, and in fact there are no color dictionary items relating to panel highlight colors. These should be structured exactly the same as the node colors, with panel-highlight, panel-highlight--alt-1, panel-highlight--alt2, etc.

Reply:
Right now, if you put some invalid name in, it depends on the component to have a default, I think. So nodes default to the "neon-coral" for example if you put a bad value in.

I think panel highlights are leaning on edge-alt-[num] palette definitions right now.